sklearn/manifold/mds.py

```diff
@@ -351,10 +352,15 @@ class MDS(BaseEstimator):
     "Multidimensional scaling by optimizing goodness of fit to a nonmetric
     hypothesis" Kruskal, J. Psychometrika, 29, (1964)
 
+    "Out-of-sample extensions for lle, isomap, mds, eigenmaps, and spectral
+    clustering." Bengio, Y. et al. Advances in neural information processing
+    systems 16 (2004): 177-184.
+
     """
     def __init__(self, n_components=2, metric=True, n_init=4,
                  max_iter=300, verbose=0, eps=1e-3, n_jobs=1,
-                 random_state=None, dissimilarity="euclidean"):
+                 random_state=None, dissimilarity="euclidean",
+                 extendible=False):
```

Comment:
*"extensible" or "extendable"
Perhaps the correct term is "inductive"

Please document the parameter in the class docstring.

Reply:
I will switch over to using "inductive".